README.md

```diff
@@ -1,6 +1,6 @@
 # Siddarth Krishnan . Jarvis Consulting
 
-I'm a recent computer science graduate passionate about machine learning and advances in the crypto space like decentralized finance. My excitement stems from seeing how we can identify patterns in big data set with the use of deep learning and different neural network architectures. In addition, I've been paying close attention to the proliferation of decentralized protocols in blockchain technology and how creative developers have gamified finance with Web3. There are many challenges I find interesting here like incentivizing liquidity providers. I would enjoy working in a data engineering or science role and learning where I can apply my math aptitude and pattern recognition skills.
+I'm a recent computer science graduate passionate about machine learning and advances in the crypto space like decentralized finance. My excitement stems from seeing how we can identify patterns in big data sets using deep learning and different neural network architectures. In addition, I've been paying close attention to the proliferation of decentralized protocols in blockchain technology and how creative developers have gamified finance with Web3. There are many challenges I find interesting here like incentivizing liquidity providers and boostrapping tokens with emissions. I would enjoy working in a data engineering or science role and learning where I can apply my math aptitude and pattern recognition skills.
```

Comment:
I'm a recent computer science graduate passionate about software development, machine learning, and advances in the crypto space like decentralized finance. My excitement stems from seeing how we can identify patterns in big data sets using deep learning and different neural network architectures. In addition, I've been paying close attention to the proliferation of decentralized protocols in blockchain technology and how creative developers have gamified finance with Web3. There are many challenges I find interesting here like incentivizing liquidity providers and boostrapping tokens with emissions. I would enjoy working in a data engineering or science role and learning where I can apply my math aptitude and pattern recognition skills.